src/main/resources/hudson/plugins/jira/JiraSite/config.jelly

```diff
@@ -28,13 +28,13 @@
     <c:select />
   </f:entry>
   <f:entry title="${%Connection timeout}" field="timeout">
-    <f:textbox default="10" />
```

Comment:
not related should be part of an other pr

Reply:
Separate PR indeed, not related to JCASC compatibility.